design-system/src/_css/main.css

```diff
@@ -48,3 +47,5 @@
 @import "design-system/swatches.css";
 @import "design-system/states.css";
 @import "design-system/footer.css";
+
+@import "@coopdigital/foundations-layout";
```

Comment:
Same comment as before -- why does this need to be imported at this stage rather than alongside the other foundation packages? Seems a bit fishy!